### PR TITLE
NOISSUE - Make event sourcing optional

### PR DESCRIPTION
**Pull request by anovakovic01**
### What does this do?
Add the env var flag for event sourcing.

### Which issue(s) does this PR fix/relate to?
There is no issue for this PR.

### List any changes that modify/break current functionality
There are no such changes. There are only additional env vars.

### Have you included tests for your changes?
No, I've tested it manually.

### Did you document any new/modified functionality?
Yes, I've updated the existing documentation.

### CODE REVIEW

**Review by manuio — APPROVED** [review]
LGTM

File: .env [hunk 1]
```diff
@@ -61,6 +61,7 @@ MF_THINGS_DB_USER=mainflux
 MF_THINGS_DB_PASS=mainflux
 MF_THINGS_DB=things
 MF_THINGS_SECRET=secret
+MF_THINGS_ES_ACTIVE=false
```
Comment (drasko) [inline .env:64]: Do we need an extra ENV var?

Can't we say: if you did not provide value for ES server URL and/or if it is an empty string - just consider that ES is not active.

**Review by drasko — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (drasko) [reply .env:64]: After chat with @anovakovic01 it was concluded that empty strings can be confusing and all other defaults changes in that case (for consistency).

So we stick with this approach.

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**